src/plugins/drawing/objects/line.tsx

```diff
@@ -82,8 +82,8 @@ export const LineObject = StrokedObject.named("LineObject")
       const height = bbox.se.y - bbox.nw.y;
       const newWidth  = width -  deltas.left + deltas.right;
       const newHeight = height - deltas.top + deltas.bottom;
-      const widthFactor = newWidth/width;
-      const heightFactor = newHeight/height;
+      const widthFactor = width ? newWidth/width : 1;
+      const heightFactor = height ? newHeight/height : 1;
```

Comment:
As we were discussing, this might fix the core problem. I looked more closely at the history event from when the nulls showed up and sure enough the two points had a dx and dy of 0. So I'd assume the bounding box would have a width and height of 0. 

If you go back to earlier in the history maybe you can figure out how the dx and dy was set to 0. I'd guess they weren't initially 0 but somehow they got scaled down to 0.